src/neuroconv/datainterfaces/ecephys/baserecordingextractorinterface.py

```diff
@@ -300,6 +303,7 @@ def add_to_nwbfile(
         nwbfile: NWBFile,
         metadata: Optional[dict] = None,
         stub_test: bool = False,
+        stub_test_size: int = 100,
```

Comment:
I would also recommend calling this `stub_frames` or anything else to indicate it is not stubbing channels along the way (I know it's in the docstring, but if it's in the variable name that's even clearer)

Reply:
We use this already, don't we?

https://github.com/catalystneuro/neuroconv/blob/74689fdfce22ad70d01e273dc5830354820b3f4e/src/neuroconv/datainterfaces/ophys/baseimagingextractorinterface.py#L109-L110